modules/images/dominant-color/hooks.php

```diff
@@ -236,7 +236,9 @@ function dominant_color_filter_content_tags( $content, $context = null ) {
  */
 function dominant_color_add_inline_style() {
 	$handle = 'dominant-color-styles';
-	wp_register_style( $handle, false );
+	// PHPCS ignore reason: Dominant color add single line inline style that does not change frequently,
+	// so the version number is not used.
```

Comment:
We can simplify this to one line. 
```suggestion
	// PHPCS ignore reason: version not used since this handle is only registered for adding an inline style.
```